deployments/typesMillau.json

```diff
@@ -0,0 +1,52 @@
+{
+	"AuthoritySet": {
+		"authorities": "AuthorityList",
+		"set_id": "SetId"
+	},
+	"BridgedBlockHash": "H256",
+	"BridgedBlockHasher": "BlakeTwoAndKeccak256",
```

Comment:
Do we not need to define `BlakeTwoAndKeccak256` for Apps?

Reply:
So when i provide this exact json file to polkadotjs , there are no errors in the console. So i assumed this was needed, but i can dive more into it.

Reply:
I think we don't even need the `Hasher` at all, given that the `BlakeTwoAndKeccak256` is for sure not defined anywhere and it doesn't complain it means it's not needed.